appserver/jdbc/jdbc-ra/jdbc-core/src/main/java/fish/payara/jdbc/stats/FrequentSQLTraceCache.java

```diff
@@ -88,7 +90,8 @@ public void checkAndUpdateCache(SQLTrace cacheObj) {
                 if (cache.size() < maxStoredEntries){
                     cache.put(cacheObj.getQueryName(), cacheObj);
                 } else {
-                    _logger.log(Level.WARNING, "Frequent SQL Trace Cache full, {0} not stored", cacheObj.getQueryName());
+                    _logger.log(Level.INFO, "Frequent SQL Trace Cache full, enable CONFIG log level for more info.");
```

Comment:
I think you need an if check on levels otherwise you will get two log messages when you enable CONFIG